tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -222,5 +254,61 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 				return releaseCreated.IsReleased()
 			}, releaseCreationTimeout, defaultInterval).Should(BeTrue())
 		})
+
+		It("validates that imageIds was pushed to pyxis are identical to those in task create-pyxis-image.", func() {
+			Eventually(func() bool {
+				isImageExist := 0
+				for _, imageId := range imageIDs {
+
+					url := fmt.Sprintf("https://pyxis.preprod.api.redhat.com/v1/images/id/%s", imageId)
+
+					// Create a TLS configuration with the key and certificate
+					cert, err := tls.X509KeyPair(pyxisCertDecoded, pyxisKeyDecoded)
+					if err != nil {
+						fmt.Println("Error creating TLS certificate and key:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					// Create a client with the custom TLS configuration
+					client := &http.Client{
+						Transport: &http.Transport{
+							TLSClientConfig: &tls.Config{
+								Certificates: []tls.Certificate{cert},
+							},
+						},
+					}
+
+					// Send GET request
+					request, err := http.NewRequest("GET", url, nil)
+					if err != nil {
+						fmt.Println("Error creating GET request:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					response, err := client.Do(request)
+					if err != nil {
+						fmt.Println("Error sending GET request:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					defer response.Body.Close()
+
+					// Read the response body
+					body, err := io.ReadAll(response.Body)
+					if err != nil {
+						fmt.Println("Error reading response body:", err)
+						Expect(err).NotTo(HaveOccurred())
+					}
+
+					// Print the response
+					if strings.Contains(string(body), imageId) {
```

Comment:
Here we should read fields in the json directly - using `json.Unmarshal()`. And then check the name is correct and then also verify that there are some contentManifestComponents in the object (at least 1).

Reply:
Done